auth/src/main/java/com/firebase/ui/auth/ui/accountlink/WelcomeBackIdpPrompt.java

```diff
@@ -64,7 +65,7 @@ public static Intent createIntent(
             Context context,
             FlowParameters flowParams,
             User existingUser,
-            IdpResponse newUserResponse) {
+            @Nullable IdpResponse newUserResponse) {
```

Comment:
This was only used to get the login credential of another idp so we had another crash where the `IdpResponse.Builder#build()` would throw an `IllegalStateException` because the email clients didn't have a `token`. So double ouch! 😱😆